app/templates/govuk_frontend_jinja_overrides/templates/components/language-select/macro.html

```diff
@@ -0,0 +1,43 @@
+{% macro hmrcLanguageSelect(params) %}
+{% if params.language == 'en' %}
+  <nav class="govuk-body hmrc-language-select" aria-label="Language switcher">
+    <ul class="hmrc-language-select__list">
+      <li class="hmrc-language-select__list-item">
+        <span aria-current="true">English</span>
+      </li>
+      <li class="hmrc-language-select__list-item">
+        <a href="{{ params.cy.href }}"
+          hreflang="cy"
+          lang="cy"
+          rel="alternate"
+          class="govuk-link"
+          data-journey-click="link - click:lang-select:Cymraeg">
+          <span class="govuk-visually-hidden">Newid yr iaith ir Gymraeg</span>
+          <span aria-hidden="true">Cymraeg</span>
+        </a>
+      </li>
+    </ul>
+  </nav>
+{% endif %}
+
+{% if params.language == 'cy' %}
```

Comment:
Should this be an elif ?

Doesn't really matter since it can't be both cy and en but just makes it easier to read

Reply:
I agree - and I can change it - but the macro above is purely based on HMRC's language select component (i.e., what it would be if we were importing the HMRC design system - but we're not)